symengine/polys/basic_conversions.cpp

```diff
@@ -134,13 +134,13 @@ class PolyGeneratorVisitorPow : public BaseVisitor<PolyGeneratorVisitorPow>
         // needs `expand` to have been called
         RCP<const Number> mulx = one, divx = one;
 
-        if (x.coef_->is_negative())
+        if (x.get_coef()->is_negative())
             mulx = minus_one;
```

Comment:
Here is an example that is correct with returning a `RCP <> &`, because it is used on the same line. So this usage is fine.